pkg/gen/input/workflows/internal/file/create_release_pr.yaml.template

```diff
@@ -114,6 +122,9 @@ jobs:
     env:
       architect_flags: "--organisation ${{ github.repository_owner }} --project ${{ needs.gather_facts.outputs.repo_name }}"
     steps:
+      - uses: actions/setup-go@v3
+        with:
+          go-version: '=1.17.0'
```

Comment:
Why not 18?

Reply:
We are using `1.17` in architect and most of our repositories specify 1.17 on their `go.mod` files, so I figure we would want to upgrade everything to 1.18 at the same time.